### DISCUSSION TITLE
I Need Starlette > 0.27.x for a custom Jinja2 environment

### DISCUSSION THREAD

**Discussion by bnmnetp** [fastapi/fastapi#9937] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
import jinja2
from fastapi.templating import Jinja2Templates

env = jinja2.Environment(...)
templates = Jinja2Templates(env=env)
```

### Description

From the command line execute the above.   It fails.  Although the fastapi docs send you to starlette for more details, this is copied exactly from the starlette docs.

### Operating System

macOS

### FastAPI Version

I have tried with 0.95.2 and 0.100.x

### Python Version

3.10, 3.11

**Comment by Kludex** [comment, marked as answer]
There's already a PR for the bump.

**Reply by Kludex** [reply]
https://github.com/tiangolo/fastapi/pull/9636

**Reply by bnmnetp** [reply]
Thanks, I don't know how I missed that.  But that is just a bump to 0.28 when the most recent release is 0.31.  I'll have to do some digging to see which version I would need.  I would offer to try to do a PR but it looked like the bump to 0.28 was pretty complicated.
